lib/png/png.go

```diff
@@ -0,0 +1,137 @@
+package png
+
+import (
+	"bytes"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	_ "embed"
+
+	"github.com/playwright-community/playwright-go"
+	"oss.terrastruct.com/d2/lib/xmain"
+)
+
+type Playwright struct {
```

Comment:
Taking suggestions for a better name for this struct

Reply:
seems fine. maybe `PlaywrightContext`